### PR TITLE
Updates to jss/javaparser

### LINKED ISSUE #475: jss sharpsign quote for static field not working
Trying the example given from JSS readme:

```
CL-USER(1): #"{java.lang.System}.{fileSeparator}"
#<THREAD "interpreter" {3B1EA186}>: Debugger invoked on condition of type UNBOUND-VARIABLE
  The variable JAVA.LANG.SYSTEM is unbound.
Restarts:
  0: TOP-LEVEL Return to top level.
```

Also, it seems that the `javaparser` dependency is quite ancient. Not sure if it should be bumped.

### PR DESCRIPTION
**Pull request by alanruttenberg**
Update to javaparser 3.24.2. Fix tests. Fix typo: ignoreable -> ignorable. Have invoke load javaparser first time #1 is used. Fixes #475 #477 #478

### CODE REVIEW

**Review by easye — APPROVED** [review]
I haven't had time to test this yet.  It would be best to add as part of the CI.  I will likely merge this in the next day unless someone reports problems.

File: contrib/jss/javaparser-tests.asd [hunk 1]
```diff
@@ -1,3 +1,4 @@
+(in-package :asdf)
```
[the remark this answers is not in the document]
Comment (alanruttenberg) [reply contrib/jss/javaparser-tests.asd:1]: Don't like relying on magic. I ran and fixed the tests.

**Comment by easye** [comment]
@alanruttenberg Please don't close the ticket until the associated pull makes it in, as it will confuse people.  Yes, it is fixed for you, but not with respect to the dev tree.  

And thanks as always for the love (and JSS, which seems to be getting some wider use these days)!

**Comment by alanruttenberg** [comment]
I closed one of 3 after I put the actionable issues in the other two tickets. Glad it's merged now.

**Comment by easye** [comment]
My mistake.  Apologies for the false statment on my part.

**Pull request merged**